packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js

```diff
@@ -159,21 +159,13 @@ export default class LocusInfo extends EventsScope {
 
   /**
    * @param {Object} locus
-   * @param {Meeting} meeting
    * @returns {undefined}
    * @memberof LocusInfo
    */
-  initialSetup(locus, meeting) {
```

Comment:
As Colin pointed out, this method is only being called without a meeting param now, so I removed the extra logic.

Reply:
@chburket  let have a 1:1 on this i feel that we will miss out something so let make sure we test the large meeting scenario